src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -148,7 +166,7 @@ const DEFAULTS = {
         zoomPoint: ZoomPoint.CURSOR,
         dragPanning: true,
         spectralMatchingType: SpectralType.VRAD,
-        autoWCSMatching: WCSMatchingType.NONE,
+        autoWCSMatching: [],
```

Comment:
Would using JS Set() be easier to manage?

Reply:
Thanks, it sounds like a good suggestion, I will look at the JS Set(). However, this code is outdated, I should keep ```autoWCSMatching``` as a number for the backward compatible purpose.